app-admin/pinkybar/metadata.xml

```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Please look at [GLEP 66](https://www.gentoo.org/glep/glep-0066.html) on how to write correct commit messages. Alternatively, look at `git log` in the repo to learn the practical way.